packages/amplify_analytics_pinpoint/android/build.gradle

```diff
@@ -48,7 +48,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
     implementation 'com.amplifyframework:aws-analytics-pinpoint:1.0.0'
-    implementation 'com.amplifyframework:aws-auth-cognito:1.0.0'
+    implementation 'com.amplifyframework:aws-auth-cognito:1.5.0'
```

Comment:
Do we need to update all other dependencies to 1.5.0 as well?

Reply:
Should these not be 1.6.3 as in other parts of the project (DataStore)? The Android team does not generally recommend mixing and matching versions in a consuming project.

Reply:
Done.